tasks/windows.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
+  [String]
+  $Action
+)
+
+$ErrorActionPreference = 'Stop'
+
+function Invoke-ServiceAction($Service, $Action)
+{
+  $inSyncStatus = 'in_sync'
+  $status = $null
+
+  switch ($Action)
+  {
+    'start'
+    {
+      if ($Service.Status -eq 'Running') { $status = $InSyncStatus }
+      else { Start-Service $Service }
+    }
+    'stop'
+    {
+      if ($Service.Status -eq 'Stopped') { $status = $InSyncStatus }
```

Comment:
What about if it's in StopPending?

Reply:
Same comment as https://github.com/puppetlabs/puppetlabs-service/pull/27#discussion_r177525854